db/migrate/20170704165711_add_user_bio_and_token.rb

```diff
@@ -0,0 +1,23 @@
+class AddUserBioAndToken < ActiveRecord::Migration
+  def up
+    add_column :rusers, :bio, :text
+    add_column :rusers, :token, :string
+    add_column :rusers, :status, :integer, default: 0
+    remove_column :rusers, :location_privacy
+
+    # copy bios into new fields for non-spam users
+    DrupalUsers.where('status != 0').each do |u|
+      user = u.user
+      user.status = u.status
```

Comment:
@jywarren okay, I am setting up plots2 again and will do this hopefully by today.